distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -47,7 +51,8 @@
     private static Action getHeaders() {
         return composite(contentType("text/plain"),
                 header("Transfer-Encoding", "chunked"),
-                header("content-type", "text/plain"));
+                header("content-type", "text/plain"),
+                header("Accept-Ranges", "bytes"));
```

Comment:
Moved it into TestFederation as a separate header method alongside getCswRetrievalHeaders() so it can be added in only when needed.